### PR TITLE
New package: Canon.IJPrinterAssistantTool version 1.90.3

### PR DESCRIPTION
**Pull request by Teknoshark**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/192555)

### CODE REVIEW

**Comment by Teknoshark** [comment]
c-wss.com domain is controlled by Canon. Source: https://www.whois.com/whois/c-wss.com

**Comment by stephengillie** [comment]
Would this be a good `PackageUrl`? 
https://www.canon-europe.com/support/consumer_products/software/ij-printer-assistant-tool.html?softwareid=tcm:13-1871303&language=EN&productid=tcm:13-2267884&os=Windows%2011

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

Should this be a nested installer?

File: manifests/c/Canon/IJPrinterAssistantTool/1.90.3/Canon.IJPrinterAssistantTool.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Canon.IJPrinterAssistantTool
+PackageVersion: 1.90.3
+InstallerType: portable
```
Comment (Trenly) [inline manifests/c/Canon/IJPrinterAssistantTool/1.90.3/Canon.IJPrinterAssistantTool.installer.yaml:6]: Based on the screenshot Stephen shared, at the very least, this needs to be `exe`

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**